dev-lua/lua-openssl/lua-openssl-0.7.3-r1.ebuild

```diff
@@ -39,6 +39,11 @@ src_unpack() {
 	popd > /dev/null || die
 }
 
+src_prepare() {
+	PATCHES=("${FILESDIR}/0001-fix-libressl-compat.patch")
```

Comment:
I'm sorry I didn't look closely at this. The normal practice is to move `PATCHES` to global var, and then you don't need to define `src_prepare` (but please move it over phase functions). Alternatively, if  you prefer keeping it inside `src_prepare`, please make it `local`.

Reply:
old habits. fixed now.